packages/engine/Source/Shaders/PostProcessStages/AcesTonemappingStage.glsl

```diff
@@ -5,6 +5,7 @@ in vec2 v_textureCoordinates;
 #ifdef AUTO_EXPOSURE
 uniform sampler2D autoExposure;
 #endif
+uniform float exposure;
```

Comment:
I would put this in a `else` block. Generally, we want to be conscientious of the number of uniforms we add.